.travis.yml

```diff
@@ -51,17 +51,6 @@ jobs:
         on:
           branch: master
 
-# Use travis to cache a Bazel disk-cache.
-# This connects to the docker container via a volume mount (`-v /home/travis/disk-cache:/disk-cache/`).
-# Bazel will store artifacts here due to the `--disk_cache=/disk-cache/` argument.
-cache:
-  npm: true
```

Comment:
I thought this was saving us a lot of time, does docker not use it? The comment suggests that it is used.

Reply:
The comment is out of date. 

While this comment refers to the bazel caches, the stuff also applies to the npm cache. 

I agree with Cameron, though, that these may not be necessary; we should experimentally verify.